src/SourceBuild/content/eng/finish-source-only.proj

```diff
@@ -207,115 +168,8 @@
-          Condition="'@(PrebuiltFile)' != '' and '$(SkipErrorOnPrebuilts)' != 'true'">
-    <Error Text="@(PrebuiltFile->Count()) Prebuilts Exist" />
-  </Target>
-
-  <Target Name="GetInputsOutputForCreatePrivateSourceBuiltArtifactsArchive"
-          DependsOnTargets="DetermineSourceBuiltSdkNonStableVersion;ResolveProjectReferences">
-    <!-- Inputs: Packages to include in the tarball -->
-    <ItemGroup>
-      <ArtifactsPackageToBundle Include="$(ArtifactsShippingPackagesDir)**;
-                                          $(ArtifactsNonShippingPackagesDir)**" />
-      <ArtifactsPackageToBundle Remove="@(ArtifactsPackageToBundle)"
-                                Condition="$([System.String]::Copy('%(Identity)').EndsWith('.symbols.nupkg'))" />
-      <ReferencePackageToBundle Include="$(ReferencePackagesDir)**" />
-      <ReferencePackageToBundle Remove="@(ReferencePackageToBundle)"
-                                Condition="$([System.String]::Copy('%(Identity)').EndsWith('.symbols.nupkg'))" />
-      <MergedAssetManifest Include="$(MergedAssetManifestOutputPath)" />
-    </ItemGroup>
-
-    <PropertyGroup>
-      <!-- Create a layout directory for the files that are to be included in the artifacts tarball. -->
-      <SourceBuiltLayoutDir>$([MSBuild]::NormalizeDirectory('$(BaseIntermediateOutputPath)', 'artifacts-layout'))</SourceBuiltLayoutDir>
 
-      <!-- Outputs -->
-      <SourceBuiltTarballName>$(ArtifactsAssetsDir)$(SourceBuiltArtifactsTarballName).$(SourceBuiltSdkNonStableVersion).$(TargetRid)$(ArchiveExtension)</SourceBuiltTarballName>
-      <SourceBuiltVersionName>$(SourceBuiltLayoutDir).version</SourceBuiltVersionName>
-      <AllPackageVersionsPropsName>$(SourceBuiltLayoutDir)PackageVersions.props</AllPackageVersionsPropsName>
-      <SourceBuiltMergedAssetManifestName>$(SourceBuiltLayoutDir)%(MergedAssetManifest.Filename)%(MergedAssetManifest.Extension)</SourceBuiltMergedAssetManifestName>
-    </PropertyGroup>
-  </Target>
-
-  <!-- Discover the produced packages from the merged asset manifest -->
-  <UsingTask TaskName="Microsoft.DotNet.UnifiedBuild.Tasks.GetKnownArtifactsFromAssetManifests" AssemblyFile="$(MicrosoftDotNetUnifiedBuildTasksAssembly)" TaskFactory="TaskHostFactory" />
-  <Target Name="GetProducedPackages" Returns="@(ProducedPackage)">
-    <GetKnownArtifactsFromAssetManifests AssetManifests="$(MergedAssetManifestOutputPath)">
-      <Output TaskParameter="KnownPackages" ItemName="ProducedPackage" />
-    </GetKnownArtifactsFromAssetManifests>
+    <Error Text="@(PrebuiltFile->Count()) Prebuilts Exist" Condition="'$(SkipErrorOnPrebuilts)' != 'true'" />
   </Target>
-
-  <!-- Create the SourceBuilt.Private.Artifacts archive when building source-only. -->
-  <UsingTask TaskName="Microsoft.DotNet.UnifiedBuild.Tasks.WritePackageVersionsProps" AssemblyFile="$(MicrosoftDotNetUnifiedBuildTasksAssembly)" TaskFactory="TaskHostFactory" />
-  <Target Name="CreatePrivateSourceBuiltArtifactsArchive"
```

Comment:
@jkoritzinsky where is this and many other source-build related targets now? I don't see them in your PR?

Reply:
The ones that aren't in this file are in [`PublishSourceBuild.props`](https://github.com/dotnet/sdk/pull/47076/files#diff-aaa58a5aa540c37ea44b79429f17a271e6b6d0c8592b21252c77c4e96a3cc975)